test.js

```diff
@@ -0,0 +1,53 @@
+const accessSync = require('fs').accessSync
```

Comment:
Using `const` here means we can't run the tests on older node. a nitpick but I deliberately avoided ES2015 features to keep old node uspport.